### ISSUE TITLE
gpu

### ISSUE THREAD

**Issue by ghost** [Trotts/Siamese-Neural-Network-MNIST-Triplet-Loss#2]
i am running your code but i can not fix the instalation part of gpu 
from tensorflow.keras.utils import multi_gpu_model

could you please help

**Comment by Trotts** [comment]
Hi, 

Can you confirm you're running in an environment with the following versions of packages installed: 

numpy==1.19.0
tensorflow==1.14.0
tensorflow-estimator==1.14.0
tensorflow-gpu==1.14.0
matplotlib==3.0.3
scikit_learn==0.24.1
